pay-queue/tests/conftest.py

```diff
@@ -183,96 +175,53 @@ def restart_savepoint(sess2, trans):  # pylint: disable=unused-variable
-    if os.getenv('TEST_NATS_DOCKER'):
-        docker_services.start('nats')
-        time.sleep(2)
-    # TODO get the wait part working, as opposed to sleeping for 2s
-    # public_port = docker_services.wait_for_service("nats", 4222)
-    # dsn = "{docker_services.docker_ip}:{public_port}".format(**locals())
-    # return dsn
-
-
 @pytest.fixture(scope='session', autouse=True)
 def auto(docker_services, app):
     """Spin up docker containers."""
     if app.config['USE_DOCKER_MOCK']:
         docker_services.start('minio')
         docker_services.start('proxy')
         docker_services.start('paybc')
-        docker_services.start('nats')
-
-
-@pytest.fixture(scope='function')
-@pytest.mark.asyncio
-async def stan(event_loop, client_id):
-    """Create a stan connection for each function, to be used in the tests."""
-    nc = Nats()
-    sc = Stan()
-    cluster_name = 'test-cluster'
-
-    await nc.connect(io_loop=event_loop, name='entity.filing.tester')
-
-    await sc.connect(cluster_name, client_id, nats=nc)
-
-    yield sc
-
-    await sc.close()
-    await nc.close()
-
-
-@pytest.fixture(scope='function')
-@pytest.mark.asyncio
-async def events_stan(app, event_loop, client_id):
-    """Create a stan connection for each function.
-
-    Uses environment variables for the cluster name.
-    """
-    nc = Nats()
-    sc = Stan()
-
-    await nc.connect(io_loop=event_loop)
-
-    cluster_name = os.getenv('STAN_CLUSTER_NAME', 'test-cluster')
-
-    if not cluster_name:
-        raise ValueError('Missing env variable: STAN_CLUSTER_NAME')
-
-    await sc.connect(cluster_name, client_id, nats=nc)
-
-    yield sc
-
-    await sc.close()
-    await nc.close()
-
-
-@pytest.fixture(scope='function')
-def future(event_loop):
-    """Return a future that is used for managing function tests."""
-    _future = asyncio.Future(loop=event_loop)
-    return _future
-
-
-@pytest.fixture
-def create_mock_coro(mocker, monkeypatch):
-    """Return a mocked coroutine, and optionally patch-it in."""
-
-    def _create_mock_patch_coro(to_patch=None):
-        mock = mocker.Mock()
-
-        async def _coro(*args, **kwargs):
-            return mock(*args, **kwargs)
 
-        if to_patch:  # <-- may not need/want to patch anything
-            monkeypatch.setattr(to_patch, _coro)
-        return mock, _coro
 
-    return _create_mock_patch_coro
+@pytest.fixture(autouse=True)
+def mock_queue_auth(mocker):
+    """Mock queue authorization."""
+    mocker.patch('pay_queue.external.gcp_auth.verify_jwt', return_value='')
 
 
 @pytest.fixture()
 def mock_publish(monkeypatch):
     """Mock check_auth."""
-    monkeypatch.setattr('pay_api.services.queue_publisher.publish', lambda *args, **kwargs: None)
+    monkeypatch.setattr('pay_api.services.gcp_queue_publisher.publish_to_queue', lambda *args, **kwargs: None)
+
+
+# @pytest.fixture(scope='session', autouse=True)
+# def initialize_pubsub(app):
```

Comment:
we should create a ticket to get this going again , we can worry about it later

Reply:
@ochiu Jia was having some issues with this, I recommended she just comment it out and we'll fix it later. 

I didn't include this in the other projects because it's fairly complex. I went the request route. Same with Names.

Reply:
Sounds good